public/partials/membership_card.html

```diff
@@ -18,7 +18,7 @@ <h3 class="white">Membership</h3>
         <div flex layout="row">
             <md-icon class="no-margin" md-svg-icon="/images/icons/icons_farmer_active.svg"></md-icon>
             <div layout-align="center" style="text-align: center;" layout="column" ng-class="{'md-offset': 1}">
-                Active
+                {{isActive(memberships[0].mi_expiration.split("T")[0])?  "Active" :  'Inactive'}}
```

Comment:
@nick22891 as discussed, since a farmer may have more than one membership returned fromt the database, accessing the array of membership using a specific index will prevent other memberships to be shown. Do a ng-repeat to resolve the issue.